ui/apps/platform/src/Containers/Vulnerabilities/utils/sortUtils.tsx

```diff
@@ -40,10 +59,19 @@ export function getWorkloadSortFields(entityTab: WorkloadEntityTab): string[] {
  * @param entityTab The chosen entity
  * @returns The default sort option
  */
-export function getDefaultWorkloadSortOption(entityTab: WorkloadEntityTab): SortOption {
+export function getDefaultWorkloadSortOption(
+    entityTab: WorkloadEntityTab,
+    searchFilter?: SearchFilter
+): SortOption | NonEmptyArray<SortOption> {
     switch (entityTab) {
         case 'CVE':
-            return { field: 'CVSS', aggregateBy: aggregateByCVSS, direction: 'desc' };
+            // Array.prototype.map does not currently retain the arity of an input tuple, so
```

Comment:
Since severity filters can be applied when the user visits the page, we need to filter the default sort option to match the applied filters.